src/_gettsim/transfers/rente.py

```diff
@@ -914,22 +1400,62 @@ def ges_rente_vorauss_frauen(
         weiblich
         and ges_rente_wartezeit_15
         and y_pflichtbeitr_ab_40 > ges_rente_params["rente_für_frauen_pflichtbeitr_y"]
-        and geburtsjahr < ges_rente_params["abolishment_cohort_rente_für_frauen"]
+        and geburtsjahr < ges_rente_params["first_birthyear_without_rente_für_frauen"]
     )
 
     return out
 
 
-def _ges_rente_vorauss_arbeitsl(
+@dates_active(end="2006-12-31", change_name="_ges_rente_vorauss_arbeitsl")
+def _ges_rente_vorauss_arbeitsl_ohne_2007_reform(
+    arbeitsl_1y_past_585: bool,
+    ges_rente_wartezeit_15: bool,
+    pflichtbeitr_8_in_10: bool,
+) -> bool:
+    """Eligibility for Altersrente für Arbeitslose (pension
+    for unemployed).
+
+    Wartezeit 15 years, 8 contribution years past 10 years, being unemployed for at
```

Comment:
Just wondering: Is there really no requirement of being *currently* unemployed? So I could be unemployed at age 59 for a year and claim at age 63 (or whatever) despite being currently employed?

If there is such a requirement, we might always think of the current implementation as an approximation (can always quit / get myself fired at any time), but then we should document that better.

Reply:
According to law, the function should look something like this:

```python
def _ges_rente_vorauss_arbeitsl(
    arbeitsl_1y_past_585: bool,
    ges_rente_wartezeit_15: bool,
    pflichtbeitr_8_in_10: bool,
    arbeitssuchend: bool,
    m_altersteilzeitarb_52: bool
) -> bool:

    return (arbeitssuchend and arbeitsl_1y_past_585 and ges_rente_wartezeit_15 and arbeitsl_1y_past_585) or (m_altersteilzeitarb_52 and arbeitsl_1y_past_585 and ges_rente_wartezeit_15 and arbeitsl_1y_past_585)

def m_altersteilzeitarb_52(
    m_altersteilzeitarb: bool
) -> bool:
     return m_altersteilzeitarb>=52
```
(of course with numbers in params files, etc)

This would require a new input variable: m_altersteilzeitarb (Months of Altersteilzeitarbeit).

But I agree, that `arbeitssuchend` is something that the potential retiree can easily influence. A quick search regarding voluntary unemployment didn't yield any restrictions. (But, of course, this doesn't guarantee that there aren't any.)
I added a few sentences in the docstring:
```text
The person is also required to be unemployed at the time of claiming the pension. As there are no restrictions regarding voluntary unemployment this requirement may be viewed as always satisfied and is therefore not included when checking for eligibility.
``